docker/autoware-universe/Dockerfile

```diff
@@ -48,6 +48,11 @@ RUN chmod 644 /etc/vulkan/icd.d/nvidia_icd.json
 ADD "https://gitlab.com/nvidia/container-images/opengl/raw/5191cf205d3e4bb1150091f9464499b076104354/glvnd/runtime/10_nvidia.json" /etc/glvnd/egl_vendor.d/10_nvidia.json
 RUN chmod 644 /etc/glvnd/egl_vendor.d/10_nvidia.json
 
+## Register OpenCL GPU vendors
+RUN mkdir -p /etc/OpenCL/vendors \
+  && echo "libnvidia-opencl.so.1" > /etc/OpenCL/vendors/nvidia.icd \
+  && chmod 644 /etc/OpenCL/vendors/nvidia.icd
```

Comment:
Could you show me some official reference links for this instruction?

Reply:
https://github.com/KhronosGroup/OpenCL-Docs/blob/a897dfb2e238e7e0b623dd9b774e67a922fe52dc/ext/cl_khr_icd.asciidoc#icd-loader-vendor-enumeration-on-linux

Reply:
Thanks! Added to the description.
Adding it as a comment would also be good. I'll leave it to you two.